packages/chat-core/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "@nlxai/chat-core",
   "version": "0.1.34",
   "description": "NLX Chat SDK core",
-  "main": "lib/index.esm.js",
-  "module": "lib/index.esm.js",
-  "commonjs": "lib/index.cjs.js",
+  "main": "lib/index.cjs",
```

Comment:
I'm not sure how broad the compatibility is here, but if it's compatible, I'd suggest we roll this out to the rest of the packages. 

also note: main is cjs